notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    probs_test = calibrated.predict_proba(testx)[:, 1]
</pre><p>What is the testing accuracy/precision/recall/f1 for this method? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
We have not calculated it, but we can add that.